src/program.js

```diff
@@ -251,6 +251,12 @@ Example: $0 --help run.
         demand: false,
         type: 'boolean',
       },
+      'pref': {
+        describe: 'Launch firefox with custom preferences. Lightweight ' +
+                  'alternative to creating custom profile.',
+        demand: false,
```

Comment:
we should also add `requiresArg: true` here 
(http://yargs.js.org/docs/#requiresArg).